tests/test_kernel_tag.py

```diff
@@ -0,0 +1,8 @@
+import unittest
+
+from oar.core.advisory import Advisory
```

Comment:
I think we don't have to add a new test module here, just need a func in `test_advisory.py`